### ISSUE TITLE
proper description on how to combine code::\noteOn, \noteOff, \polyTouch in Reference/MKtl_description_files

### ISSUE THREAD

**Issue by LFSaw** [ModalityTeam/Modality-toolkit#131]
depends on #130

**Comment by adcxyz** [comment]
an idea for this: an ElementGroup consisting of 2 or 3 elements, one each for \noteOn and \noteOff, and maybe polyTouch could be recognized as one semantic unit, and one gui. 
In the proposed new notation scheme, something like:
(
// ...,
elementsDesc: (
	// 8 pads with noteOn/noteOff
	(36..43).collect { |num, i| 
		(
			shared: (elementType: \pad, midiChan: 0, midiNum: num, spec: \midiNum),
			elements: (
				noteOn: (midiMsgType: \noteOn, ioType: \inout),
				noteOff: (midiMsgType: \noteOff, ioType: \inout),
				polyTouch: (midiMsgType: \polyTouch,ioType: \in),
			)
		)
	}
);
)

**Comment by adcxyz** [comment]
Btw, the same thing goes for joysticks - an elementsDesc with a joyAxis \x, joyAxis \y and and a \hatSwitch could auto-create an ElementGroup that knows how to draw itself as a joystickgui with 3 elements, as suggested in #217. roughly like this:
```
(
(
// ...,
elementsDesc: (
	joy: (
		shared: (gui: \StickView),
		elements: (
			x: (elementType: \joyAxis, ioType: \in, spec: \hidAxis),
			y: (elementType: \joyAxis, ioType: \in, spec: \hidAxis),
			hat: (elementType: \joyHat, ioType: \in, spec: \hidBut),
		),
	)
)
);
)

```

**Comment by adcxyz** [comment]
closed in desc_reorg with faad7e0
